client/src/app/panel/tabs/linting/LintingTab.less

```diff
@@ -45,10 +45,27 @@
   }
 
   .linting-tab-item__content {
-    display: flex;
     width: calc(100% - 200px);
     padding: var(--linting-tab-item-padding);
 
+    .linting-tab-item__link {
+      margin-left: 0.4em;
+      margin-right: 0.4em;
+      cursor: default;
```

Comment:
Any reason why we would hide accessible cursor for links?

```suggestion
```

Reply:
We're not consistent with cursors. We keep the default cursor for most clickable elements. 🤔

Reply:
Let's distinguish links from buttons. Links more or less consistently show the default ("hand") cursor: 

![capture Y1yLtC_optimized](https://github.com/camunda/camunda-modeler/assets/58601/b7d7be14-f2b0-4c6b-9b64-85343e8debc2)